stoppable_workers_test.go

```diff
@@ -1,4 +1,4 @@
-package utils_test
```

Comment:
Sorry -- I wrote my test calling `NewStoppableWorkerWithTicker` and saw red when getting the compile error. I didn't need to do this, but it was already done.

I don't think* there's a concern with using the non `_test` package here.